.github/workflows/run-metrics.yaml

```diff
@@ -30,3 +30,15 @@ jobs:
 
       - name: run launch times on clh
         run:  bash tests/metrics/gha-run.sh run-test-launchtimes-clh
+
+      - name: run memory foot print on qemu
+        run:  bash tests/metrics/gha-run.sh run-test-memory-usage-qemu
+
+      - name: run memory foot print on clh
+        run:  bash tests/metrics/gha-run.sh run-test-memory-usage-clh
+
+      - name: run memory usage inside container qemu
+        run:  bash tests/metrics/gha-run.sh run-test-memory-usage-inside-container-qemu
+
+      - name: run memory usage inside container clh
+        run:  bash tests/metrics/gha-run.sh run-test-memory-usage-inside-container-clh
```

Comment:
All of those steps could be made into a matrix, similar to what we have done here: https://github.com/kata-containers/kata-containers/blob/19890133e9b40ebe74d4f7359e61f56bedcb65ee/.github/workflows/run-k8s-tests-on-aks.yaml#L19-L28

This could simplify the process and make the code duplication smaller.

Let's **not** change it right now, though, but please, make this change in a future PR, okay?

Reply:
@fidencio I take the opportunity to add the matrix in this PR.